tests/textlab_api/textlab_api_test.py

```diff
@@ -0,0 +1,302 @@
+
+    # Enable Textlab in the test region
+    Region.objects.filter(slug="augsburg").update(hix_enabled=True)
+
+    edit_page, response = update_page_content(
+        admin_client, "Willkommen in Augsburg", ""
+    )
+
+    assert response.status_code == 302
+    assert response.headers.get("Location") == edit_page
+
+    # Check that the HIX score has not been updated
+    page_translation = Page.objects.get(id=2).get_translation("de")
+
+    assert page_translation.hix_score is None
+    assert mock_server.requests_counter == 0
+
+
+@pytest.mark.django_db
+def test_hix_no_content_changes(
+    load_test_data: None,
+    admin_client: Client,
+    settings: SettingsWrapper,
+    mock_server: MockServer,
+) -> None:
+    """
+    Check that the HIX score is copied from the previous translation version if the page content is not changed. No request to TextLab API is expected.
+
+    :param load_test_data: The fixture providing the test data (see :meth:`~tests.conftest.load_test_data`)
+    :param admin_client: The fixture providing the http client
+    :param settings: The fixture providing the django settings
+    :param mock_server: The fixture providing the dummy http server
+    """
+
+    # Setup a mocked Textlab API server with dummy responses
+    mock_server.configure("/user/login", 200, {"token": "dummy"})
+    mock_server.configure("/benchmark/420", 500, {"message": "Error occurred"})
+
+    # Redirect call aimed at the Textlab API to the fake server
+    settings.TEXTLAB_API_URL = f"http://localhost:{mock_server.port}"
+
+    # Enable Textlab in the test region
+    Region.objects.filter(slug="augsburg").update(hix_enabled=True)
+
+    # Setting HIX score initial value
+    initial_hix_score = 11
+    PageTranslation.objects.filter(page__id=2, language__slug="de").update(
+        hix_score=initial_hix_score
+    )
+
+    # Check initial page properties
+    page_translation = Page.objects.get(id=2).get_translation("de")
+    content_before_change = page_translation.content
+    assert content_before_change != ""
+    assert page_translation.latest_version.hix_score == initial_hix_score
+
+    edit_page, response = update_page_content(
+        admin_client, "Neuer Titel", content_before_change
+    )
+
+    assert response.status_code == 302
+    assert response.headers.get("Location") == edit_page
+
+    # Check that the HIX score has been copied from the previous version
+    page_translation = Page.objects.get(id=2).get_translation("de")
+
+    assert page_translation.hix_score == initial_hix_score
+    assert page_translation.content == content_before_change
+    assert mock_server.requests_counter == 0
+
+
+@pytest.mark.django_db
+def test_hix_response_400(
+    load_test_data: None,
+    admin_client: Client,
+    settings: SettingsWrapper,
+    mock_server: MockServer,
+) -> None:
+    """
+    Check that the HIX score is requested but response 400 received. Hix score is not updated
```

Comment:
What do you think about this wording? I think it's a bit clearer.

```suggestion
    Check that the HIX score is not being updated when the API returns a HTTP 400 error.
```